lib/api/auth_user.dart

```diff
@@ -28,41 +29,53 @@ class Auth {
   }
 
   Future<dynamic> userAuth() async {
-    String getId = await fetchId();
-    String token= await fetchToken();
+    var arr=await fetchToken();
+    print(arr);
+
+    Map<String, String> requestHeaders = {
+      'Content-type': 'application/json',
+      'Accept': 'application/json',
+      'Authorization': accessToken
+    };
 
     final res = await http.get(
-        Uri.parse('http://161.35.99.225/api/v1/users/$getId'),
-        headers: {HttpHeaders.authorizationHeader: token,});
+        Uri.parse('http://161.35.99.225/api/v1/users/$id'),
+        headers:requestHeaders);
 
-    return res.body.toString();
-  }
 
-  Future fetchId() async {
-    final res = await http.post(Uri.parse(baseUrl),
-        headers: <String, String>{'Authorization': basicAuth});
-    var jsonData = jsonDecode(res.body);
-    id = jsonData['result']['id'].toString();
-    if (res.statusCode == 200) {
-      return id;
-    } else {
-      throw Exception('Failed to fetch id');
-    }
-  }
+    print('----------'+res.body);
 
+    return res.body;
+  }
 
+  // Future fetchId() async {
+  //   final res = await http.post(Uri.parse(baseUrl),
+  //       headers: <String, String>{'Authorization': basicAuth});
+  //   var jsonData = jsonDecode(res.body);
+  //   print(jsonData);
+  //   if (res.statusCode == 200) {
+  //     id = jsonData['result']['id'];
+  //     return id;
+  //   } else {
+  //     throw Exception('Failed to fetch id');
+  //   }
+  // }
 
   Future<dynamic> fetchToken() async {
-    String accessToken='wrong token';
+    String basicAuth =
+        'Basic ${base64.encode(utf8.encode('$username:$password'))}';
```

Comment:
You are passing empty string for both `username` and `password`. I hardcoded credentials and it worked.